t/utils/utils.go

```diff
@@ -70,7 +70,7 @@ func init() {
 	flag.Parse()
 
 	// set up logger
-	if err := logutils.OverrideRootLog(*logLevel, "", true); err != nil {
+	if err := logutils.OverrideRootLog(true, *logLevel, "", true); err != nil {
```

Comment:
Constant for enabling or extension of comment that's always enabled during tests would be nice.